packages/fetchai/connections/p2p_libp2p/utils/utils.go

```diff
@@ -50,6 +50,10 @@ import (
 	"libp2p_node/aea"
 )
 
+const (
```

Comment:
Why this (particular) bound?

Reply:
it is arbitrary considering the regular messages that agents currently exchange

Reply:
We should keep track of this choice in docs